backend/src/main/java/org/eclipse/tractusx/puris/backend/erp_adapter/controller/ErpAdapterController.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2024 Volkswagen AG
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.eclipse.tractusx.puris.backend.erp_adapter.controller;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.swagger.v3.oas.annotations.Parameter;
+import lombok.extern.slf4j.Slf4j;
+
+import org.eclipse.tractusx.puris.backend.erp_adapter.logic.service.ItemStockErpAdapterService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Date;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("erp_adapter")
+@Slf4j
+public class ErpAdapterController {
+
+    @Autowired
+    private ItemStockErpAdapterService itemStockErpAdapterService;
+
+    @PutMapping
+    public ResponseEntity<?> putMethod(
+        @RequestParam("request_id") UUID requestId,
```

Comment:
I checked the naming conventions for web apps. At least following this [post](https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris), we should go for dashes instead of underscores. Likely we did it already differently somewhere else :/

Reply:
Done